### ISSUE TITLE
Correlate goals with the product

### ISSUE THREAD

**Issue by richtabor** [Automattic/wp-calypso#95490]

**Comment by nuriapenya** [comment]
👋 heya! Isn't this issue a repetition of https://github.com/Automattic/wp-calypso/issues/95496 ?

**Comment by autumnfjeld** [comment]
Yes, @nuriapenya  it seems like a duplicate.  So let's close this since we have these two:

[Goals & Launchpad Tasks #95496](https://github.com/Automattic/wp-calypso/issues/95496)

[Correlate Goals with plugins and WordPress configuration #95491](https://github.com/Automattic/wp-calypso/issues/95491)
